src/main/kotlin/br/unipar/plano/domain/procedimento/model/Procedimento.kt

```diff
@@ -0,0 +1,121 @@
+package br.unipar.plano.domain.procedimento.model
+
+import br.unipar.plano.domain.carteirinha.model.Carteirinha
+import br.unipar.plano.domain.carteirinha.model.StatusCarteirinha
+import br.unipar.plano.domain.contrato.model.Contrato
+import br.unipar.plano.domain.prestador.model.Prestador
+import br.unipar.plano.domain.solicitacaoprocedimento.service.SolicitacaoProcedimentoQueryService
+import java.math.BigDecimal
+import java.time.LocalDate
+import java.util.*
+import javax.persistence.*
+
+@Entity
+class Procedimento(
+
+        @field:EmbeddedId
+        val id: IdProcedimento,
+
+        @Column(nullable = false)
+        val dataEmissao: LocalDate,
+
+        @OneToOne(cascade = [CascadeType.ALL], orphanRemoval = true)
```

Comment:
Você pode ter vários procedimento para um contrato. O mesmo vale para o prestador e carteirinha.

```suggestion
        @ManyToOne(cascade = [CascadeType.ALL], orphanRemoval = true)
```